test/index.js

```diff
@@ -1,11 +1,36 @@
-/* eslint-disable no-console */
+const ava = require('ava');
 const snekfetch = require('../index');
+const TestServer = require('./server');
+const testServer = new TestServer();
```

Comment:
this could probably be improved to `cosnt testServer = new (require("./server"))()`